README.md

```diff
@@ -5,6 +5,10 @@ Math magicians is a website for all fans of mathematics. It is a Single Page App
 
 This project is part of the Microverse curriculum Module 3 Block 1. The main objective is about building your web application using React.
 
+![screenshot](img/pic1.png)
```

Comment:
- [ ] Please deploy the project online and include the live link in the readme.

Reply:
Okay I deployed